moto/identitystore/models.py

```diff
@@ -0,0 +1,28 @@
+"""IdentityStoreBackend class with methods for supported APIs."""
+import uuid
```

Comment:
Please use `moto.moto_api._internal.mock_random` instead. That will guarantee that UUID's are always deterministic when Moto is seeded.

Reply:
Gr8

Changed accordingly